draft-gpew-priv-ppm.md

```diff
@@ -519,22 +524,18 @@ struct {
 This message is called the client's *report*. It contains the following fields:
 
 * `task_id` is the task ID of the task for which the report is intended.
-* `time` is the time at which the report was generated. This field is used by
-  the aggregators to ensure the report appears in at most one batch. (See
-  {{anti-replay}}.)
-* `nonce` is a random number chosen by the client generating the report. This
-  and the timestamp field are used by the aggregators to ensure that each report
-  appears at most once in a batch. (See {{anti-replay}}.)
+* `nonce` is a nonce generated by the client and used by the aggregators to
+  uniquely identify the report in the context of a PPM task. The nonce includes
+  the time at which the report was generated and a random 64-bit integer. This
```

Comment:
nit: since the `Nonce` type appears in a few places, I think the discussion of its `time` and `rand` members should be moved to the definition of `struct Nonce`, around line 417.